src/graphql/taskList.graphql.ts

```diff
@@ -0,0 +1,22 @@
+import gql from 'graphql-tag';
```

Comment:
It looks like all previous files in `graphql` folder have naming convention for file name in Kebab/Spinal case.
And this's Camel case, so please or update all previous file namings to Camel case or change current files names

Reply:
True, we are naming files always with camel case, why only this folder is in Kebab/Spinal?

Reply:
I have no idea, that's wierd - so let's update it to use everywhere camel case 😉

Reply:
Hmmm..🤔 Well I guess different files types follows different conventions. We of course can use camelCase for `graphql/*.graphql.ts` like `newbieSelect.graphql.ts` (which is only non kebab-case file in directory), honestly I'm not a big fan of using React convention for _non React_ files. Is it something that Apollo suggest? 

Not only graphql files were written with kebab-case. Utils files also used same convention

![image](https://user-images.githubusercontent.com/14184100/69725089-9f25cd80-111d-11ea-924d-5cefefa076a4.png)

Reply:
Also `graphql/*.graphql.ts` have consts defined as upper case snake case `UPPER_CASE_SNAKE` not camelCase. 

For me I would use kebab case everywhere except React's component, Context etc.. I mean this is most common way for naming files.

Reply:
We will follow michal suggestion